authors/apps/authentication/models.py

```diff
@@ -117,3 +117,25 @@ def get_short_name(self):
         the user's real name, we return their username instead.
         """
         return self.username
+
+    @property
+    def token(self):
+        """
+        Allows us to get a user's token by calling `user.token` 
+
```

Comment:
Great work @charitymarani, I like the function here that allows us to get the token from different parts of our project, however, kindly remove the whitespaces to adhere to pep8 standards

Reply:
Working on it